src/pages-helpers/pupil/options-pages/options-pages-helpers.ts

```diff
@@ -1,22 +1,21 @@
 import { GetStaticPropsContext, GetStaticPropsResult } from "next";
-import { groupBy } from "lodash";
 import {
   ProgrammeFields,
   examboardSlugs,
+  pathwaySlugs,
 } from "@oaknational/oak-curriculum-schema";
 
 import { PupilProgrammeListingData } from "@/node-lib/curriculum-api-2023/queries/pupilProgrammeListing/pupilProgrammeListing.schema";
 import OakError from "@/errors/OakError";
 import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
 import { resolveOakHref } from "@/common-lib/urls";
-import { getAvailableProgrammeFactor } from "@/components/PupilViews/PupilProgrammeListing/getAvailableProgrammeFactor";
-import { ExamboardData } from "@/components/PupilComponents/BrowseExamboardSelector";
-import { TierData } from "@/components/PupilComponents/BrowseTierSelector";
+import { getAvailableProgrammeFactor } from "@/pages-helpers/pupil/options-pages/getAvailableProgrammeFactor";
 import { PupilViewsProgrammeListingProps } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
 
 export type OptionsURLParams = {
   programmeSlug: string;
   examboardSlug?: string;
+  pathwaySlug?: string;
```

Comment:
we need this for when we navigate back from the unit listing page of a programme with EB and pathway